src/transformers/modeling_tf_pytorch_utils.py

```diff
@@ -319,7 +320,13 @@ def load_pytorch_state_dict_in_tf2_model(
                     continue
             raise AttributeError(f"{name} not found in PyTorch model")
 
-        array = apply_transpose(transpose, pt_state_dict[name], symbolic_weight.shape)
+        try:
+            array = apply_transpose(transpose, pt_state_dict[name], symbolic_weight.shape)
+        except tf.errors.InvalidArgumentError:
+            if not ignore_mismatched_sizes:
+                raise
+            else:
+                continue
```

Comment:
We should keep track of those in a `mismatched_keys` list and return it at the end of this function, so it can be shown as a warning at the end (that's what we do for standard load).

Reply:
Added only with `output_loading_info` (to respect pre-existing code).